package.json

```diff
@@ -25,7 +25,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "mocha -r ts-node/register src/utils/test.ts",
+    "test": "mocha -r ts-node/register src/utils/getLargestPathLengthInLevelTest.ts",
```

Comment:
Why did you edit the test? How will you test other files now, you have to change this everytime? @sahandfarahani  @Manpreet-Bhatti

Reply:
@ralph-dev I'll change it back to react-scripts test but yeah essentially when testing with mocha you have to use that script every time.